__tests__/channels.test.ts

```diff
@@ -1,96 +1,151 @@
-import { describe, expect, test } from 'vitest'
-import { ReaderInstance } from '../src/reader'
-import { one, uri, client, logger } from '../src/testUtils'
+import { describe, expect, test, vi } from 'vitest'
+import { StreamMsgMock } from '../src/testUtils'
+import { WriterInstance } from '../src/writer'
+import { FromRunner } from '@rdfc/proto'
+import winston, { createLogger } from 'winston'
+import { StreamIdentify } from '@rdfc/proto/lib/generated/common'
```

Comment:
This should probably just be imported from `'@rdfc/proto'` as well?

Reply:
Correct